src/main/java/org/gitlab4j/api/ProtectedBranchesApi.java

```diff
@@ -155,13 +155,14 @@ public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
      */
     public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
             AccessLevel pushAccessLevel, AccessLevel mergeAccessLevel, AccessLevel unprotectAccessLevel,
-            Boolean codeOwnerApprovalRequired) throws GitLabApiException {
+            Boolean codeOwnerApprovalRequired, Boolean allowForcedPush) throws GitLabApiException {
```

Comment:
Usually when we add a new parameter we also add a method with the old signature (using java method overloading) to preserve backward compatibility.